toolset/utils/docker_helper.py

```diff
@@ -44,6 +44,55 @@ def clean(benchmarker_config):
                 client.images.remove(image.id, force=True)
     client.images.prune()
 
+def __build(base_url, path, build_log_file, log_prefix, dockerfile, tag):
+    '''
+    Builds the dependency chain as well as the test implementation docker images
+    for the given tests.
+    '''
```

Comment:
There is no such thing as a 'dependency chain' anymore.

Reply:
Ah yeah, that was a copy/pasta comment.

Reply:
I figured. Change it with a `[ci skip]` and you got my approval.